tests/tests/drupal/check-deployed.yaml

```diff
@@ -2,7 +2,7 @@
   hosts: localhost
   serial: 1
   vars:
-    url: "http://nginx-php.{{ project | regex_replace('_', '-') }}.{{ branch | regex_replace('/', '-') }}.{{ lookup('env','OPENSHIFT_ROUTE_SUFFIX') }}"
```

Comment:
can we keep the nginx-php test and just test varnish additionally?

Reply:
I think having a default varnish secret and then overriding with EP and a secret in the project would work. My only worry is that a client forgets to have that secret set and ends up using the default secret in production, bit of a security risk there.

I'll get going on the other requested changes.